invenio_pages/records/models.py

```diff
@@ -36,6 +37,9 @@ class PageModel(db.Model, Timestamp):
     title = db.Column(db.String(200), nullable=False, default="")
     """Page title."""
 
+    lang = db.Column(db.String(2), nullable=False, default="en")
```

Comment:
minor/question: (probably I18N/L10N experts can jump in) is this the best format to store the language? E.g. could `en-US` or similar be better and map to the overall translations process?

Reply:
I'm not an expert but this is the information the UI provides?! This follows the settings in `I18N_LANGUAGES`.